lib/views/commit-view.js

```diff
@@ -299,6 +358,13 @@ export default class CommitView extends React.Component {
   toggleCoAuthorInput() {
     this.setState({
       showCoAuthorInput: !this.state.showCoAuthorInput,
+    }, () => {
+      if (this.state.showCoAuthorInput) {
+        this.refCoAuthorSelect.focus();
```

Comment:
:sparkles: Nice. I didn't even know I wanted this to happen until I saw you two making it happen 😆